plugin/federation/federation.gotpl

```diff
@@ -6,6 +6,12 @@
 
 {{ reserveImport "github.com/99designs/gqlgen/plugin/federation/fedruntime" }}
 
+var (
+	ErrUnknownType = errors.New("unknown type")
+	ErrTypeNotFound = errors.New("type not found")
+	ErrUnableToResolveEntity = errors.New("unable to resolve Entity")
```

Comment:
Is ErrUnableToResolveEntity being used?  It seems like it isn't...

Reply:
Not any more :)

Reply:
yay!